packages/server-renderer/src/__tests__/index.test.ts

```diff
@@ -3,18 +3,23 @@ import {
   FeatureServiceBinder,
   FeatureServiceProviderDefinition
 } from '@feature-hub/core';
+// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
Its a test file, you can disable this rule for the whole file like in the other test files:

// tslint:disable:no-implicit-dependencies